specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json

```diff
@@ -117,6 +130,56 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Subscription/PurchaseSupport": {
+      "post": {
+        "operationId": "PurchaseSupportPlan_Post",
+        "x-ms-examples": {
+          "PurchaseSupport": {
+            "$ref": "./examples/PurchaseSupport.json"
+          }
+        },
+        "description": "The operation to purchase/convert/cancel support plan.",
+        "parameters": [
```

Comment:
This API might feel more intuitive if it were a PUT (create or update) and a DELETE (delete) on the supportPlan resource. What's the reason to model it as a POST?